scss/components/_messages.scss

```diff
@@ -18,39 +24,41 @@
 .message {
   @include clearfix();
   background-size: 2rem;
-  background-color: $ivory;
+  background-color: #eee6dd;
```

Comment:
Color literals like `#eee6dd` should only be used in variable declarations; they should be referred to via variable everywhere else.